composer.json

```diff
@@ -15,8 +15,7 @@
   "require": {
     "php": ">=5.4",
     "google/apiclient": "^2.0",
-    "guzzlehttp/guzzle": "~5.3.3",
-    "johnpbloch/wordpress-core": "^5.1"
```

Comment:
@ataylorme we still need this package as `required-dev` for visual regression testing.

Reply:
@ivankristianto for the AMP plugin @westonruter is testing against a site on Pantheon. I would be happy to help you set that up as well for Site Kit so you do not need to run WordPress inside your CI instance.

I can help with migrating Visual Regression tests to that workflow as well (I spoke about them at WordCamp Europe 😁).

Reply:
Either way, if you want to start a branch with `johnpbloch/wordpress-core` in `require-dev` instead of `require` I am happy to test that also